tests/test_common.py

```diff
@@ -69,3 +73,24 @@ def test_lockfile_retain_status_by_default():
     assert lf.acquire(blocking=False)[0] is True
     assert lf.read_status() == 'some_status_1'
     lf.release()
+
+def test_parse_settings():
+    res = parse_settings(["number:12", "string:somthing", "bool:false"])
+    assert res == {'bool': False, 'number': 12, 'string': 'somthing'}
+
+    res = parse_settings(["nested.number:12", "nested.string:somthing", "nested.bool:false"])
+    assert res == {'nested': {'bool': False, 'number': 12, 'string': 'somthing'}}
+
+    # a fix that need to be backported from upstream
```

Comment:
how about filing an issue?

Reply:
I've pasted there the commit, we don't really need it for anything (at least not yet)

Reply:
okay, don't feel strong here. just thought that an issue is easier to track than a comment in code which is not even marked with `FIXME` or `TODO`.